test/FunctionalTests/Microsoft.OData.Edm.Tests/Csdl/CsdlReaderJsonTests.cs

```diff
@@ -1858,6 +1858,41 @@ public void ParsingPropertyWithUntypedTypeAsPropertyTypeInJsonWorks()
             Assert.Equal("Collection(Edm.Untyped)", dataProperty.Type.FullName());
         }
 
+        [Fact]
+        public void ReadJsonKeyAlias()
+        {
+            var csdl = @"{
+""$Version"": ""4.0"",
+""$EntityContainer"": ""NS1.Container"",
+  ""NS1"": {
+    ""Product"": {
+      ""$Kind"": ""EntityType"",
+      ""$Key"": [
+        {
+          ""ExampleObjectName"": ""NS1.ExampleComplexObject/ExampleObject""
+        }
+      ],
+      ""ExampleObject"": {
+        ""$Type"": ""NS1.ExampleComplexObject""
+      },
+      ""Name"": {},
+      ""UpdatedTime"": {
+        ""$Type"": ""Edm.Date""
+      }
+    },
+    ""Container"": {
+      ""$Kind"": ""EntityContainer"",
+      ""Products"": {
+        ""$Collection"": true,
+        ""$Type"": ""NS1.Product""
+      }
+    }
+  }
+}"
+;
+            IEdmModel model = Parse(csdl);

```

Comment:
Could you add assertions to verify that the model has been successfully read? Could you demonstrate the key aliases being retrieved and the key properties being inspected?